packages/pn-pa-webapp/src/pages/components/NewNotification/PreliminaryInformations.tsx

```diff
@@ -60,7 +60,7 @@ const PreliminaryInformations = ({ notification, onConfirm }: Props) => {
 
   const validationSchema = yup.object({
     paProtocolNumber: requiredStringFieldValidation(tc, 256),
-    subject: requiredStringFieldValidation(tc, 134),
+    subject: requiredStringFieldValidation(tc, 134).min(10, tc('too-short-field-error', { minLength: 10 })),
```

Comment:
Can you put the min check into requiredStringFieldValidation function as done for max?

Reply:
Ciao @AndreaCimini90 I preferred to avoid that since `subject` is the *only* field including a `min` check, while `requiredStringFieldValidation` is used ~10 times.

Reply:
Even when I don't agree, I will proceed as suggested given @AndreaCimini90 still thinks it's the best thing to do.